config/config.hpp.in

```diff
@@ -125,6 +125,10 @@
 // Requires a CUDA compiler (nvcc).
 // #define MFEM_USE_CUDA
 
+// Build the GPU/HIP-enabled version of the MFEM library.
```

Comment:
This could maybe say "AMD GPU/HIP-enabled"

Reply:
done